.gitignore

```diff
@@ -12,3 +12,5 @@ dist
 .tox
 build/
 .coverage*
+.vscode
```

Comment:
I also use the amazing vscode, but would rather keep editor-specific things out of the `.gitignore` file - any chance you could remove this line and add it to your global `.gitignore` file, as described here https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/ ?